apps/machine/zynqmp_r5/platform_info.c

```diff
@@ -229,6 +229,13 @@ int platform_poll(void *priv)
 
 	prproc = rproc->priv;
 	while(1) {
+#ifdef RPMSG_NO_IPI
+		char * done_polling = (char*)POLL_SHM_LOCATION;
```

Comment:
please rework your both patches using proper coding style.  you can use https://github.com/OpenAMP/open-amp/pull/187 to help you.

Reply:
good point. redone so that it is using metal_io_region instead

Reply:
coding style still wrong:

.```
/scripts/checkpatch.pl --strict -g HEAD-2
------------------------------------------------------------------------------
Commit c6408e33a8ab ("apps: zynqmp_r5: add switch to run r5 demo without IPI")
------------------------------------------------------------------------------
c6408e33a8ab0edcd22dd936cee3c30bac4f08a0:45: ERROR:POINTER_LOCATION: "(foo*)" should be "(foo *)"
#45: FILE: apps/machine/zynqmp_r5/platform_info.c:61:
+			.virt = (void*)POLL_BASE_ADDR,

c6408e33a8ab0edcd22dd936cee3c30bac4f08a0:90: WARNING:BLOCK_COMMENT_STYLE: Block comments use * on subsequent lines
#90: FILE: apps/machine/zynqmp_r5/platform_info.c:241:
+/*
+		char * done_polling = (char*)POLL_SHM_LOCATION;

c6408e33a8ab0edcd22dd936cee3c30bac4f08a0:96: ERROR:SPACING: space required before the open brace '{'
#96: FILE: apps/machine/zynqmp_r5/platform_info.c:247:
+		if (metal_io_read32(prproc->poll_io, 0)){

c6408e33a8ab0edcd22dd936cee3c30bac4f08a0:198: WARNING:LONG_LINE: line over 80 characters
#198: FILE: apps/machine/zynqmp_r5/zynqmp_r5_a53_rproc.c:70:
+	ret = metal_device_open(prproc->poll_dev_bus_name, prproc->poll_dev_name,

c6408e33a8ab0edcd22dd936cee3c30bac4f08a0:265: CHECK:PARENTHESIS_ALIGNMENT: Alignment should match open parenthesis
#265: FILE: apps/machine/zynqmp_r5/zynqmp_r5_a53_rproc.c:173:
+	metal_io_write32(prproc->poll_io, IPI_TRIG_OFFSET,
 			  prproc->ipi_chn_mask);

```

Reply:
updated as per comments.